crates/next-custom-transforms/tests/fixture/server-actions/server/56/output.js

```diff
@@ -0,0 +1,14 @@
+/* __next_internal_action_entry_do_not_use__ {} */ import { registerServerReference } from "private-next-rsc-server-reference";
+import { encryptActionBoundArgs, decryptActionBoundArgs } from "private-next-rsc-action-encryption";
+import { cache as $$cache__ } from "private-next-rsc-cache-wrapper";
```

Comment:
this could be optimized later when there's no usage then we don't insert?

Reply:
This is handled in #73160.